### PR TITLE
Add authall redirects and front auth popup controls

### PR DESCRIPTION
**Pull request by lempiy**

### CODE REVIEW

File: djTaskLine/dj_app/dj_main/settings/base.py [hunk 1]
```diff
@@ -36,6 +36,8 @@
 ACCOUNT_EMAIL_REQUIRED = True
 ACCOUNT_AUTHENTICATION_METHOD = 'email'
 ACCOUNT_USERNAME_REQUIRED = False
+# ACCOUNT_ADAPTER = 'apps.users.adapter.CustomAccountAdapter'
+# LOGIN_REDIRECT_URL = '/'
```
Comment (pogorelov-ss) [inline djTaskLine/dj_app/dj_main/settings/base.py:40]: default redirect
 or use ?next=/

File: djTaskLine/dj_app/apps/users/adapter.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+class CustomAccountAdapter(DefaultAccountAdapter):
+
+    def get_login_redirect_url(self, request):
+        path = "/"
+        return path
+
+    # def get_sign_redirect_url(self, request):
+    #     path = "/"
+    #     return path
```
Comment (pogorelov-ss) [inline djTaskLine/dj_app/apps/users/adapter.py:12]: unneded

File: djTaskLine/dj_app/apps/user_profiles/templates/user_profiles/user-profile-temporary.html [hunk 3]
```diff
@@ -1,3 +1,8 @@
 {% load account %}
 
-{% user_display user %}
\ No newline at end of file
+{% user_display user %}
+<script type="text/javascript">
+
+    window.close();
+    window.parent.location.href = "/";
+</script>
```
Comment (pogorelov-ss) [inline djTaskLine/dj_app/apps/user_profiles/templates/user_profiles/user-profile-temporary.html:8]: close window and redirect to / ... maybe we need change url :)

File: djTaskLine/dj_app/dj_main/settings/base.py (hunk 1 above)
Comment (pogorelov-ss) [reply djTaskLine/dj_app/dj_main/settings/base.py:40]: default redirect url = accounts/profile/

**Comment by lempiy** [comment]
Made default redirect on '/'. Still have problem with detecting user auth status on front.
Wanted to extend LoginView to put some additional  cookies to let frontend know that user has logged in

File: djTaskLine/dj_app/dj_main/settings/base.py (hunk 1 above)
Comment (lempiy) [reply djTaskLine/dj_app/dj_main/settings/base.py:40]: Made default redirect on '/'. Still have problem with detecting user auth status on front.
Wanted to extend LoginView to put some additional cookies to let frontend know that user has logged in

Comment (lempiy) [reply djTaskLine/dj_app/dj_main/settings/base.py:40]: Maybe smth like this
```py
class UserProfileTemporaryView(TemplateView):
    template_name = 'user_profiles/user-profile-temporary.html'

    def get(self, request, *args, **kwargs):
        response = super().get(request, *args, **kwargs)
        if request.user.is_authenticated():
            response.set_cookie('auth', "test", max_age=request.session.get_expiry_age())
        else:
            response.delete_cookie('auth')
        return response
```

**Review by pogorelov-ss — APPROVED** [review]

**Pull request merged**